test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -8550,6 +8550,7 @@ TEST_F(AtenXlaTensorTest, TestTriu) {
       AllClose(output, xla_output);
     });
   }
+  ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
```

Comment:
nit: thanks for adding this, and why don't we expect the xla counters to be changed (e.g., `  ExpectCounterChanged("xla::triu", cpp_test::GetIgnoredCounters())`)?